src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -616,6 +631,50 @@ ControlCompOutput(
 
 }
 
+bool
+BBConvergeCheck(
+	int const SimCompNum,
+	Real64 const MaxFlow,
+	Real64 const MinFlow
+)
+{
+
+	// FUNCTION INFORMATION:
+	//       AUTHOR         Rick Strand
+	//       DATE WRITTEN   November 2017
+	
+	// PURPOSE OF THIS SUBROUTINE:
+	// This is an additional check for the radiant/convective baseboard units
+	// to see if they are converged or the flow is sufficiently converged to
+	// procede with the simulation.  With the radiant component to these systems,
+	// the impact on the load met is more difficult to calculate and the impact
+	// on the actual system output is not as well behaved as for convective
+	// systems.  This additional check avoids excessive iterations and max
+	// iteration warnings and provides sufficiently converged results.  It is
+	// only called from ControlCompOutput.
+
+	// Return Value
+	bool BBConvergeCheck;
+	
+	// SUBROUTINE PARAMETER DEFINITIONS:
+	static Real64 const BBIterLimit( 0.00001 );
+
+	if ( SimCompNum != 5 && SimCompNum != 6 ) {
```

Comment:
I don't really like these magic numbers.  Are these called out as actual named enums in the equipment data modules somewhere?  Otherwise I don't understand what these are.

Reply:
@Myoldmopar Hmmm...so in ControlCompOutput, this is sort of hardwired into a local data array.  There aren't even any integer parameters.  I can tackle this two ways:
1. I can create parameters locally in my new subroutine that spell this out.
2. Maybe this can be done at the namespace level and then ControlCompOutput can use them as well.
What do you prefer?

Reply:
@Myoldmopar went with option 2 as that was a bit cleaner.  Hopefully I didn't introduce any errors in the process and this is now ready to go.